graphql2/graphqlapp/alert.go

```diff
@@ -155,6 +176,63 @@ func (a *Alert) Service(ctx context.Context, raw *alert.Alert) (*service.Service
 	return (*App)(a).FindOneService(ctx, raw.ServiceID)
 }
 
+func (a *Alert) RecentEvents(ctx context.Context, obj *alert.Alert, opts *graphql2.AlertRecentEventsOptions) (*graphql2.AlertLogEntryConnection, error) {
+	if opts == nil {
+		opts = new(graphql2.AlertRecentEventsOptions)
+	}
+
+	var s alertlog.SearchOptions
+	if opts.Limit != nil {
+		s.Limit = *opts.Limit
+	}
+	if s.Limit == 0 {
+		s.Limit = search.DefaultMaxResults
+	}
+	err := validate.Many(
+		validate.Range("Limit", s.Limit, 15, search.MaxResults),
```

Comment:
Minimum should be `1`, but we can drop this entirely as the `Search` method should do the validation already.